src/parsers/xccdf.ts

```diff
@@ -94,10 +136,10 @@ export function processXCCDF(xml: string, removeNewlines: false, useRuleId: 'gro
         control.id = rule.group['@_id']
         break;
       case 'rule':
-        if (rule['@_id'].toLowerCase().startsWith('sv')) {
-          control.id = rule['@_id'].split('r')[0]
+        if (rule['@_id'][0].toLowerCase().startsWith('sv')) {
```

Comment:
I am not sure what would be the best way to describe it, but maybe we can add a comment here to explain the purpose of indexing into the rule object with [0]

```suggestion
        // The way this content is parsed puts it in a nested array which means it needs to be indexed into to get out the desired content
        if (rule['@_id'][0].toLowerCase().startsWith('sv')) {
```